lib/exchange/sideshift/sideshift_exchange_provider.dart

```diff
@@ -36,46 +37,52 @@ class SideShiftExchangeProvider extends ExchangeProvider {
     CryptoCurrency.scrt,
     CryptoCurrency.stx,
     CryptoCurrency.bttc,
+    CryptoCurrency.usdt,
+    CryptoCurrency.eos,
   ];
 
   static List<ExchangePair> _supportedPairs() {
-    final supportedCurrencies = CryptoCurrency.all
-        .where((element) => !_notSupported.contains(element))
-        .toList();
+    final supportedCurrencies =
+        CryptoCurrency.all.where((element) => !_notSupported.contains(element)).toList();
 
     return supportedCurrencies
-        .map((i) => supportedCurrencies
-            .map((k) => ExchangePair(from: i, to: k, reverse: true)))
+        .map((i) => supportedCurrencies.map((k) => ExchangePair(from: i, to: k, reverse: true)))
         .expand((i) => i)
         .toList();
   }
 
   @override
-  ExchangeProviderDescription get description =>
-      ExchangeProviderDescription.sideShift;
+  ExchangeProviderDescription get description => ExchangeProviderDescription.sideShift;
 
   @override
   Future<double> fetchRate(
       {required CryptoCurrency from,
       required CryptoCurrency to,
-      required double amount,
+      double? amount,
       required bool isFixedRateMode,
-      required bool isReceiveAmount}) async {
+      bool? isReceiveAmount}) async {
     try {
-      if (amount == 0) {
-        return 0.0;
-      }
-      final fromCurrency = _normalizeCryptoCurrency(from);
-      final toCurrency = _normalizeCryptoCurrency(to);
-      final url =
-          apiBaseUrl + rangePath + '/' + fromCurrency + '/' + toCurrency;
+      final fromCurrency = from.title.toLowerCase();
+      final toCurrency = to.title.toLowerCase();
+      final depositNetwork = _networkFor(from);
+      final settleNetwork = _networkFor(to);
+
+      final url = apiBaseUrl +
+          rangePath +
+          '/' +
+          fromCurrency +
+          '-' +
+          depositNetwork +
+          '/' +
+          toCurrency +
+          '-' +
+          settleNetwork +
+          '?affiliateId=' +
+          affiliateId;
```

Comment:
I don't see this as a query parameter for this API in their docs

Reply:
I added this initially, but it's no longer needed after discussing with the SideShift team. It can be removed.